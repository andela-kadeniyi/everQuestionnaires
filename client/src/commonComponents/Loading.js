import React from 'react';

const Loading = () => (
    <div className="loadingScreen">
      ...Loading
    </div>
);

export default Loading;