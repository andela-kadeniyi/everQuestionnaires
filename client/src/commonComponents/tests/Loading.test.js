import React from 'react';
import { shallow } from 'enzyme';
import Loading from '../Loading';

describe('Loading component', () => {
  it('should render loading page', () => {
    const component = shallow(
      <Loading />
    );
    expect(component.text()).toContain("...Loading")
  });
});