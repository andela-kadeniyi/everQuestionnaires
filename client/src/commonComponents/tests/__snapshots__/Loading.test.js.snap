// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component should render loading page 1`] = `
<div
  className="loadingScreen"
>
  ...Loading
</div>
`;
